tests/test_vqe.py

```diff
@@ -1306,14 +1300,14 @@ def circuit1():
             return qml.expval(H1)
 
         res = qml.draw(circuit1)()
-        expected = " 0: ──H──╭Hamiltonian──┤  \n" + " 2: ─────╰Hamiltonian──┤  \n"
+        expected = " 0: ──H──╭┤ ⟨Hamiltonian(1, 1, 1)⟩ \n" + " 2: ─────╰┤ ⟨Hamiltonian(1, 1, 1)⟩ \n"
```

Comment:
Since the observable has now a parameter

Reply:
🤔

Reply:
I think while it may be slightly confusing what the 1,1,1 here are, it is important info when looking at the circuit as a parametrised circuit?

Reply:
Yeah. Now that I reflect, I think my main thought is that there is only half the information present (no Pauli terms).

But at the same time I'm not sure this is a big enough thing to worry about changing

Reply:
I was going to suggest we make an issue to update the circuit drawer to do something like :

```
 0: ──H──╭┤ ⟨2 X0 + 3 Z0 Z1⟩
 2: ─────╰┤ ⟨2 X0 + 3 Z0 Z1⟩
```

but often times Hamiltonians are huge and so this would look terrible; maybe we can do something similar to how arbitrary unitaries get printed, e.g., marked by "U0" in the circuit, and then the actually matrix is printed below the circuit?

```
 0: ──H──╭┤ ⟨Ham⟩
 2: ─────╰┤ ⟨Ham⟩

Ham = 2 X0 + 3 Z0 Z1
```

Reply:
That's a nice idea. Will open an issue!